From fd521a858b475d8278919fc2fc6079a301960fdf Mon Sep 17 00:00:00 2001
From: Ray Donnelly <mingw.android@gmail.com>
Date: Sun, 4 May 2014 22:54:41 +0100
Subject: [PATCH 1/2] change 'int size' to 'off_t size' in
 add_transaction_sizes_row

.. to fix negative display of 'Total Install Size'
---
 src/pacman/util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pacman/util.c b/src/pacman/util.c
index d42e27b..03e5a21 100644
--- a/src/pacman/util.c
+++ b/src/pacman/util.c
@@ -432,7 +432,7 @@ static void table_free(alpm_list_t *headers, alpm_list_t *rows)
 	alpm_list_free(rows);
 }
 
-static void add_transaction_sizes_row(alpm_list_t **rows, char *label, int size)
+static void add_transaction_sizes_row(alpm_list_t **rows, char *label, off_t size)
 {
 	alpm_list_t *row = NULL;
 	char *str;
-- 
1.9.2

